homeassistant/components/http/__init__.py

```diff
@@ -227,6 +227,21 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     source_ip_task = create_eager_task(async_get_source_ip(hass))
 
+    codespace_domain = os.getenv("GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN")
+    codespace_name = os.getenv("CODESPACE_NAME")
```

Comment:
I don't know if we should add vendor/dev environmental-specific code here.
I think it would make more sense if the development scripts would set this up correct in the configuration instead of adjusting code for these cases.